src/styles/typography.scss

```diff
@@ -91,3 +91,11 @@ h1, h2, h3, h4, h5, h6, p, ul {
   font-size: var(--text-size-3);
   line-height: 1.5;
 }
+
+/* FIXME Remove this once links and buttons are styled properly */
+a, button[data-store-button] {
```

Comment:
Hey @gvc, Thank for taking care of this! 
Since it's temporary, can you please move this to `pattern-library.scss` file?

Reply:
Will do this as soon as Lighthouse indicates I'm in the right direction :)